packages/components/psammead-headings/src/index.jsx

```diff
@@ -37,8 +34,8 @@ export const SubHeading = styled.h2.attrs(({ text }) => ({
   color: ${C_SHADOW};
   font-family: ${GEL_FF_REITH_SANS};
   margin: 0; /* Reset */
-  padding: ${GEL_SPACING_DBL} 0;
-  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+  padding-bottom: ${GEL_SPACING_TRPL} 0;
+  ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
```

Comment:
The issue says "up to 400px" but
```
export const GEL_GROUP_CD_MIN_WIDTH = 37.5; // 600px

LAPTOP_AND_LARGER: `@media (min-width: ${GEL_GROUP_CD_MIN_WIDTH}rem)`,
```

Reply:
Confirmed with UX IRL. The issue was wrong, I've updated